### PR TITLE
feat(active-release): Send 1 hour deploy summary notification

### PR DESCRIPTION
**Pull request by scttcper**
The notification is sent 1 hour after a deploy is finished via a task that runs every 5 minutes

email
Screen Shot 2022-07-06 at 6 00 13 PM

slack
Screen Shot 2022-07-06 at 6 00 27 PM

### CODE REVIEW

File: src/sentry/tasks/release_summary.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+from sentry.notifications.notifications.activity.release_summary import (
+    ReleaseSummaryActivityNotification,
+)
+from sentry.notifications.utils.participants import _get_release_committers
+from sentry.tasks.base import instrumented_task
+from sentry.types.activity import ActivityType
+
+
+@instrumented_task(name="sentry.tasks.digest.release_summary")
+def prepare_release_summary():
```
Comment (wedamija) [inline src/sentry/tasks/release_summary.py:15]: This needs tests

File: src/sentry/tasks/release_summary.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    """
+    Summarize new issues in a release an hour after deployment
+    """
+    now = timezone.now()
+    deploys = (
+        Deploy.objects.filter(
+            date_finished__gte=now - timedelta(hours=1),
+            date_finished__lt=now - timedelta(hours=1, minutes=5),
```
[the remark this answers is not in the document]
Comment (barkbarkimashark) [reply src/sentry/tasks/release_summary.py]: Created:
https://github.com/getsentry/sentry/pull/36499

File: src/sentry/tasks/release_summary.py [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    """
+    now = timezone.now()
+    start = now - timedelta(hours=1, minutes=5)
+    end = now - timedelta(hours=1)
+    releases = Release.objects.filter(
+        deploy__date_finished__gte=start,
+        deploy__date_finished__lt=end,
+        deploy__notified=True,
+    )
+
```
Comment (barkbarkimashark) [inline src/sentry/tasks/release_summary.py:27]: Probably me being overly cautious, but do we need to persist this summary was sent already? I know the task runs every 5 minutes, and the `start` and `end` times should cover it. Just a thought. 

For this test, we might not need it?

**Review by barkbarkimashark — APPROVED** [review]

File: tests/sentry/mail/activity/test_release_summary.py [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+        self.commit1 = self.another_commit(0, "a", self.user1, repository)
+        self.commit2 = self.another_commit(1, "b", self.user2, repository)
+
+    def test_simple(self):
+        with self.feature("organizations:active-release-notification-opt-in"):
+            release_summary = ReleaseSummaryActivityNotification(
+                Activity(
+                    project=self.project,
+                    user=self.user1,
+                    type=ActivityType.RELEASE.value,
```
Comment (barkbarkimashark) [inline tests/sentry/mail/activity/test_release_summary.py:51]: should this be `type=ActivityType.Deploy.value`?

**Review by wedamija — COMMENTED** [review]
This should not be deployed until the index is in place. Can you revert this until it's ready?

**Pull request merged**